go/kbfs/libkbfs/favorites_test.go

```diff
@@ -261,6 +277,7 @@ func TestFavoritesListFailsDuringAddAsync(t *testing.T) {
 }
 
 func TestFavoritesControlUserHistory(t *testing.T) {
+	t.Skip()
```

Comment:
Is this something you meant to address before putting up for review?

Reply:
whooops I marked all the broken ones with this first. I guess I missed one. will fix.